robotbase/src/main/java/frc/robot/subsystems/EndAffector.java

```diff
@@ -46,4 +46,8 @@ public void stop() {
   public void getMotorSpeed() {
     m_motor.get();
   }
+
+  public double getMotorVoltage() {
+    return m_motor.getBusVoltage();
```

Comment:
This returns the voltage fed into the motor controller. We want to be testing amps applied to the motor. See `getOutputCurrent()`